README.md

```diff
@@ -53,186 +53,10 @@ fully-supported.
-
-Download the latest [tarball][latest-release], and run the following:
-
-```shell
-tar -xvf <path-to-tarball> -C /usr/local/bin
-git-credential-manager configure
-```
-
-To uninstall:
-
-```shell
-git-credential-manager unconfigure
-rm $(command -v git-credential-manager)
-```
-
-#### Option 2: Install from source helper script
-
-1. Ensure `curl` is installed:
-
-   ```shell
-   curl --version
-   ```
-
-   If `curl` is not installed, please use your distribution's package manager
-   to install it.
-
-1. Download and run the script:
-
-   ```shell
-   curl -LO https://aka.ms/gcm/linux-install-source.sh &&
-   sh ./linux-install-source.sh &&
-   git-credential-manager-core configure
-   ```
-
-   **Note:** You will be prompted to enter your credentials so that the script
-   can download GCM's dependencies using your distribution's package
-   manager.
-
-To uninstall:
-
-[Follow these instructions][linux-uninstall] for your distribution.
-
-**Note:** all Linux distributions
-[require additional configuration][gcm-credstores] to use GCM.
-
----
-
-### Windows
-
-GCM is included with [Git for Windows][git-for-windows], and the latest version
-is included in each new Git for Windows release. This is the preferred way to
-install GCM on Windows. During installation you will be asked to select a
-credential helper, with GCM being set as the default.
-
-![image][git-for-windows-screenshot]
-
-#### Standalone installation
-
-You can also download the [latest installer][latest-release] for Windows to
-install GCM standalone.
-
-**:warning: Important :warning:**
-
-Installing GCM as a standalone package on Windows will forcibly override the
-version of GCM that is bundled with Git for Windows, **even if the version
-bundled with Git for Windows is a later version**.
-
-There are two flavors of standalone installation on Windows:
-
-- User (preferred) (`gcmuser-win*`):
-
-  Does not require administrator rights. Will install only for the current user
-  and updates only the current user's Git configuration.
-
-- System (`gcm-win*`):
-
-  Requires administrator rights. Will install for all users on the system and
-  update the system-wide Git configuration.
-
-To install, double-click the desired installation package and follow the
-instructions presented.
-
-#### Uninstall (Windows 10)
-
-To uninstall, open the Settings app and navigate to the Apps section. Select
-"Git Credential Manager" and click "Uninstall".
-
-#### Uninstall (Windows 7-8.1)
-
-To uninstall, open Control Panel and navigate to the Programs and Features
-screen. Select "Git Credential Manager" and click "Remove".
-
-#### Windows Subsystem for Linux (WSL)
-
-Git Credential Manager can be used with the [Windows Subsystem for Linux
-(WSL)][ms-wsl] to enable secure authentication of your remote Git
-repositories from inside of WSL.
-
-[Please see the GCM on WSL docs][gcm-wsl] for more information.
+See [this page](install) for installation instructions for your operating
+system.
```

Comment:
I wonder if we should move this (now much smaller) section above the `Current Status` section?